__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("api/topics", () => {
+  test("GET:200 returns topic data", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((topic) => {
```

Comment:
Make sure that you check for the length before doing an array method here, because the array method will not run if the length is 0 :)